tutorials/Channel.register.md

```diff
@@ -0,0 +1,31 @@
+Register an action to be called by dispatching from any channelClient or channelProvider. It is required to send back a corresponding acknowledgement (can be undefined).  Once a listener is registered for a particular action, it stays in place receiving and responding to incoming messages until it is removed.  This messaging mechanism works exactly the same when messages are dispatched from the provider to a client.  However, the provider has an additional publish method that sends messages to all connected clients.
+
```

Comment:
`It is required to send back a corresponding acknowledgement (can be undefined).`

Becomes -> the return value will be sent back as an acknowledgement to the original caller.  You can throw an error to send a negative-acknowledgement.

Reply:
.. and the error will reject the promise returned to the sender by the dispatch call.